### PR TITLE
Preparing debian release for 0.7.0

### PR DESCRIPTION
**Pull request by ddemidov**
Summary of changes:

1. Update copyright information in debian/copyright (see https://github.com/rhempel/ev3dev-lang-python/issues/207#issuecomment-250687801)
2. Fix lintian warnings (a line too long, and wrong weekday)
3. Get rid of hard-coded `__version__` in `ev3dev/core.py`, and replace it with auto-generated `ev3dev/version.py`.

### CODE REVIEW

**Comment by ddemidov** [comment]
@dlech, I followed http://www.ev3dev.org/docs/devtools/packaging-for-ev3dev/ up to the point I have `python3-ev3dev_0.7.0_all.deb` in `pbuilder-ev3dev/debian/jessie-armel/`. `lintian` seems to like the package, and I am able to install it with `dpkg -i python3-ev3dev_0.7.0_all.deb`

```py
>>> from ev3dev.version import __version__
>>> print(__version__)
```
gives me `0.7.0`. Assuming nobody objects against the changes proposed here, what are my next steps?

Do I need to generate the package for other values of `DIST` and `ARCH`? Or this does not matter since the deb name suggests it is suitable for all arches?

How do I release the package? Just send the deb to you?

**Comment by WasabiFan** [comment]
Thanks for doing this!

**Comment by ddemidov** [comment]
Can I get an approve from @dlech?

**Comment by dlech** [comment]
> armel

Since this package does not contain any binary components, you can actually replace `armel` with `amd64` and the build will run much faster.

You will also need to build a second time with `OS=raspbian ARCH=armhf DIST=jessie` to produce a package for raspberry pi.

> what are my next steps?

If you will be the uploader, you need to run `dch -r` in the source code directory to update the changelog with your name/email. You will also need to send me an SSH public key that will be used for uploading.

> How do I release the package? Just send the deb to you?

I will update http://www.ev3dev.org/docs/devtools/packaging-for-ev3dev/ shortly with more detailed steps on uploading.

**Comment by dlech** [comment]
Since this git repo has special handling of tags, you may not want to push the debian package tag. For the releases I did, I just pushed the tag to my personal fork.

**Comment by ddemidov** [comment]
Does `gbp` create annotated tags? If not, it should not affect anything. Also, the debian packaging script does not actually use tag info: it just dumps the version found in changelog into `RELEASE-VERSION`: https://github.com/rhempel/ev3dev-lang-python/blob/develop/debian/rules

**Comment by dlech** [comment]
> Does gbp create annotated tags?

no

> If not, it should not affect anything

I thought that it will update the readthedocs site.

> Also, the debian packaging script does not actually use tag info: it just dumps the version found in changelog into RELEASE-VERSION:

Yes, this is intentional.

File: ev3dev/core.py [hunk 1]
```diff
@@ -61,15 +61,12 @@
 INPUT_AUTO = ''
 OUTPUT_AUTO = ''
 
-# Keep the __version__ in sync with the package version in debian/changelog
```
Comment (WasabiFan) [inline ev3dev/core.py]: Why was this removed?

Comment (ddemidov) [inline ev3dev/core.py]: Because a4a296a creates ev3dev/version.py automatically. Less chances to forget to update this.

Comment (WasabiFan) [inline ev3dev/core.py]: Ah, I see. Missed that.
